src/NuGet.Services.AzureSearch/SearchService/IndexOperationBuilder.cs

```diff
@@ -33,7 +33,7 @@ public IndexOperation V3Search(V3SearchRequest request)
             }
 
             var text = _textBuilder.Build(parsed);
-            var parameters = _parametersBuilder.V3Search(request);
+            var parameters = _parametersBuilder.V3Search(request, IsEmptySearchQuery(text));
```

Comment:
Oh yeah, this seems best. It's acting the actual query we send to Azure Search and the text information doesn't leak into the parameter builder.